myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -49,15 +58,15 @@ az.style.use("arviz-darkgrid")
 
 # Model averaging
```

Comment:
This should be removed, it is now at the top of the file

Reply:
Done.